### ISSUE TITLE
Central UI with each cluster having its own Prometheus for metrics checking

### ISSUE THREAD

**Issue by Verhaeg** [similarweb/statusbay#215]
Currently I saw that it is possible to have one StatusBay server with multiple clientes connected to it. Each client in its own isolated cluster watching the annotations for that cluster and sending information to the central API.

I wonder if it is possible to have those clients check the metrics from the Prometheus server from its own cluster and only have a central UI to check how everything is working (or not).
Basically the idea is to fetch the metrics and send them together to the server and let the server run the calculations and save all the information.

Thanks
